torch/_dynamo/variables/builtin.py

```diff
@@ -257,6 +293,39 @@ def dynamic_handler(tx, a, b, options, fn=op):
         ]
         op_handlers[operator.add].extend(list_like_addition_handlers)
 
+        def list_iadd_handler(tx, a, b, options):
+            if not a.mutable_local or not b.has_unpack_var_sequence(tx):
+                # Handler doesn't apply
+                return None
+
+            return tx.replace_all(
+                a,
+                ListVariable(
+                    list(a.items) + list(b.unpack_var_sequence(tx)),
+                    regen_guards=False,
+                    **options,
+                ),
+            )
+
+        def tuple_iadd_handler(tx, a, b, options):
```

Comment:
It's weird that tuple iadd needs to be handled explicitly, because this should just go to tuple add

Reply:
You're right, lemme simplify